### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by katranci** [jiminny/jiminny-waveform#2]
https://github.com/jiminny/jiminny-waveform/blob/c9264b94b39ea979d70ee45aecd6935e26ec0a92/src/Helpers/Convertor.js#L3

Always use parens `()` to group the operations to improve readability

**Comment by notargetbg** [comment]
You are right, but it looked so clean, compared to other one I polluted so much... : )
